src/plugins/plugin-mapbox-gl/raster-layer-mapbox-gl.js

```diff
@@ -0,0 +1,120 @@
+import { replace } from 'utils/query';
+import { MapboxLayer } from '@deck.gl/mapbox';
+
+import TileLayer from './custom-layers/tile-layer';
+
+const getTileData = ({ x, y, z }, url) => {
+  const mapSource = url
+    .replace('{z}', z)
+    .replace('{x}', x)
+    .replace('{y}', y);
+
+  return fetch(mapSource)
+    .then(response => response.blob())
+    .then((response) => {
+      const src = URL.createObjectURL(response);
+      const image = new Image();
+
+      image.src = src;
+      return image;
+    });
+};
+
+const RasterLayer = (layerModel) => {
+  const {
+    layerConfig,
+    params,
+    sqlParams,
+    decodeParams,
+    id,
+    opacity,
+    decodeFunction
+  } = layerModel;
+
+  const layerConfigParsed = layerConfig.parse === false
+    ? layerConfig
+    : JSON.parse(replace(JSON.stringify(layerConfig), params, sqlParams));
+  let tileUrl;
+
+  const { body, url } = layerConfigParsed || {};
+  const { paint, minzoom, maxzoom } = body || {};
+
+  switch (layerModel.provider) {
+    case 'gee':
+      tileUrl = `https://api.resourcewatch.org/v1/layer/${id}/tile/gee/{z}/{x}/{y}`;
```

Comment:
Let’s find another solution. The library should be agnostic to the application/api

Reply:
I cannot remember why this is here exactly, I think currently the api configs need this to be somewhere to work with rasters.

Reply:
Just bringing up what we talked about offline: this should be removed and a better solution needs to be thought of before merging. What about a resource-watch service? That lives outside of the layer-manager, and a way to pass custom providers/services?